### PR TITLE
[Merged by Bors] - chore(Logic/Function/Defs): move file out of Init

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/DeriveTraversable.lean [hunk 1]
```diff
@@ -4,6 +4,9 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Simon Hudon
 -/
 import Mathlib.Control.Traversable.Lemmas
+import Lean.Elab.Match
```
Comment (grunweg) [inline Mathlib/Tactic/DeriveTraversable.lean:7]: I'm assuming these were transitively imported before?

**Review by grunweg — COMMENTED** [review]
One small question - otherwise, looks good to me. Thanks for doing this!
maintainer merge

File: Mathlib/Tactic/DeriveTraversable.lean (hunk 1 above)
Comment (jcommelin) [reply Mathlib/Tactic/DeriveTraversable.lean:7]: Yeah, this is the result of shake doing its thing

**Pull request closed without merging**